apps/bynder/src/index.html

```diff
@@ -16,12 +16,15 @@
       }
 
       .dialog-container {
-        height: 800px;
+        height: 600px;
+        display: flex;
+        justify-content: center;
         position: relative;
       }
     </style>
   </head>
   <body>
+    <script src="https://d8ejoa1fys2rk.cloudfront.net/5.0.5/modules/compactview/bynder-compactview-2-latest.js"></script>
```

Comment:
Wouldn't this cause the external JS to be loaded even when the modal isn't opened? Any way to avoid that?

Reply:
Good point 👍  I changed it to be loaded dynamically now